backend/search/models.py

```diff
@@ -0,0 +1,54 @@
+from django.db import models
+
+from treebanks.models import Component
+
+
+class ComponentSearchResult(models.Model):
+    xpath = models.TextField()
+    component = models.ForeignKey(Component, on_delete=models.CASCADE)
+    search_completed = models.DateTimeField(null=True, editable=False)
+    last_accessed = models.DateField(null=True, editable=False)
+    results = models.TextField(default='', editable=False)
```

Comment:
I think we should have a separate discussion about how to save/cache results, but this is ok for now

Reply:
I agree! I will work something out and then we will have a good basis for discussion.